features/sharding/distsql/parser/src/main/antlr4/imports/sharding/Keyword.g4

```diff
@@ -119,30 +119,10 @@ HINT
     : H I N T
     ;
 
-SET
-    : S E T
-    ;
-
-ADD
-    : A D D
-    ;
-
-DATABASE_VALUE
-    : D A T A B A S E UL_ V A L U E
-    ;
-
 TABLE_VALUE
```

Comment:
Hi @yx9o 
Why does TABLE_VALUE need to be preserved?

Reply:
Ok, probably missed it.